libs/api/domains/air-discount-scheme/src/lib/api-domains-air-discount-scheme.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common'
+import { AuthModule } from '@island.is/auth-nest-tools'
+import { AirDiscountSchemeClientModule } from '@island.is/clients/air-discount-scheme'
+
+import { AirDiscountSchemeResolver } from './api-domains-air-discount-scheme.resolver'
+import { AirDiscountSchemeService } from './api-domains-air-discount-scheme.service'
+
+@Module({
+  providers: [AirDiscountSchemeResolver, AirDiscountSchemeService],
+  imports: [AirDiscountSchemeClientModule, AuthModule],
+  exports: [AirDiscountSchemeService],
```

Comment:
Is some external module importing this one and using the service? Otherwise the service shouldn't be exported.

Reply:
Oh yes. Well spotted, thanks :+1: